internal/restic/tree_test.go

```diff
@@ -94,6 +94,28 @@ func TestNodeComparison(t *testing.T) {
 	rtest.Assert(t, !node.Equals(n2), "nodes are equal")
 }
 
+func TestTreeEqualSerialization(t *testing.T) {
```

Comment:
Shouldn't this test check with more than one node in the tree to get better coverage?

Reply:
Now the test adds 1 to 3 files and compares the serialization.